llvm/lib/Target/RISCV/RISCVInsertVSETVLI.cpp

```diff
@@ -757,7 +757,8 @@ class RISCVInsertVSETVLI : public MachineFunctionPass {
   bool computeVLVTYPEChanges(const MachineBasicBlock &MBB,
                              VSETVLIInfo &Info) const;
   void computeIncomingVLVTYPE(const MachineBasicBlock &MBB);
-  void emitVSETVLIs(MachineBasicBlock &MBB);
+  void emitVSETVLIs(MachineBasicBlock &MBB,
+                    SmallVectorImpl<MachineInstr *> &DeadVLInstrs);
```

Comment:
Dead here is misleading, these are only possibly dead.

Rename to something like DefsWithRemovedUses, or PossiblyDeadInstrs.